apps/namadillo/src/App/Settings/Advanced.tsx

```diff
@@ -0,0 +1,35 @@
+import { ActionButton, Input, Stack } from "@namada/components";
+import { useAtom } from "jotai";
+import { useState } from "react";
+import { rpcUrlAtom } from "slices/settings";
+
+export const Advanced = (): JSX.Element => {
+  const [currentRpc, setCurrentRpc] = useAtom(rpcUrlAtom);
+  const [rpc, setRpc] = useState(currentRpc);
+
+  const onSubmit = (e: React.FormEvent): void => {
+    e.preventDefault();
+    setCurrentRpc(rpc);
+    window.history.back();
```

Comment:
It probably doesn't matter, but navigating directly to `http://localhost:5173/settings/advanced` and changing the RPC will direct you out of the application.